src/i18n/Settings.js

```diff
@@ -117,6 +117,15 @@ export default {
     zh: '如果未加载身份验证插件，则默认情况下允许匿名身份验证。建议在生产部署中禁用该选项！',
     en: 'Allow anonymous authentication by default if no auth plugins loaded. Disable the option in production deployment',
   },
+  crl_cache_http_timeout: {
+    zh: `获取 CRL 时 HTTP 请求的超时限制。该参数适用于所有启用了 CRL 检查的监听器。`,
+    en: `The timeout for the HTTP request when fetching CRLs. This is a global setting for all listeners with CRL check enabled.`,
```

Comment:
```suggestion
    zh: '获取 CRL 时 HTTP 请求的超时限制。该参数适用于所有启用了 CRL 检查的监听器。',
    en: 'The timeout for the HTTP request when fetching CRLs. This is a global setting for all listeners with CRL check enabled.',
```

We need not use the JavaScript template syntax. Just use single quotes.